pom.xml

```diff
@@ -16,8 +16,21 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
+    <license.licenseName>MIT</license.licenseName>
   </properties>
 
+  <developers>
+    <developer>
```

Comment:
you could also add the email element and add the tracecompass mailing list email.